client/src/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,214 @@
+          marginBottom: 40,
+        }}
+      >
+        <h1 style={{ fontSize: 36, fontWeight: 'bold', margin: 0 }}>
+          Find your next soccer experience
+        </h1>
+        <p style={{ fontSize: 16, marginTop: 8 }}>
+          View upcoming events, explore personalized packages, and more
+        </p>
+      </div>
+
+      <div
+        style={{
+          backgroundColor: 'rgb(187, 187, 187)',
+          borderRadius: 50,
+          padding: '12px 24px',
+          display: 'flex',
+          gap: 12,
+          flexWrap: 'wrap',
+          justifyContent: 'center',
+          alignItems: 'center',
+          position: 'relative',
+          zIndex: 1,
+        }}
+      >
+        <Select
+          placeholder="Select Location"
+          style={{ borderRadius: 32, width: 220 }}
+          value={location}
+          onChange={(val) => setLocation(val)}
+          suffixIcon={<EnvironmentOutlined />}
+        >
+          {countries.map((c) => (
+            <Option key={c.code} value={c.name}>
+              {c.name}
+            </Option>
+          ))}
+        </Select>
+
+        <RangePicker
+          style={{ borderRadius: 5, width: 220 }}
+          onChange={setDateRange}
+          placeholder={['Start Date', 'End Date']}
+          suffixIcon={<CalendarOutlined />}
+        />
+
+        <Select
+          value={`${priceRange[0]} - ${priceRange[1]}`}
+          style={{ borderRadius: 32, width: 180 }}
+          suffixIcon={<DollarOutlined />}
+          dropdownRender={() => (
+            <div style={{ padding: 12 }}>
+              <Slider
+                range
+                min={MIN_PRICE}
+                max={5000}
+                value={priceRange}
+                onChange={(val) => setPriceRange(val as [number, number])}
+                tooltip={{ formatter: (val) => `$${val}` }}
+              />
+            </div>
+          )}
+        >
+          <Option value="budget">{`${priceRange[0]} - ${priceRange[1]}`}</Option>
+        </Select>
+
+        <Select
+          placeholder="Select Country"
+          style={{ borderRadius: 32, width: 200 }}
+          value={country}
+          onChange={(val) => {
+            setCountry(val);
+            setLeague('');
+            setTeam('');
+            setLeagueId(undefined);
+          }}
+          suffixIcon={<EnvironmentOutlined />}
+        >
+          {countries.map((c) => (
+            <Option key={c.code} value={c.name}>
+              {c.name}
+            </Option>
+          ))}
+        </Select>
+
+        <Select
+          placeholder="Select League"
+          style={{ borderRadius: 5, width: 200, backgroundColor: 'grey' }}
+          value={league}
+          onChange={(val, option: any) => {
+            setLeague(val);
+            setTeam('');
+            setLeagueId(option?.id);
+          }}
+          disabled={!country}
+          suffixIcon={<TrophyOutlined />}
+        >
+          {leagues.map((l) => (
```

Comment:
```suggestion
          {leagues.map((option) => (
```
or some other meaningful name

Reply:
6b3910de09c4c4ef0591a67be690bf044627fff2